IDEAS/Examples/Tutorial/DetailedHouse/DetailedHouse6.mo

```diff
@@ -1,6 +1,7 @@
-within IDEAS.Examples.Tutorial;
-model Example6 "Extension of example 5 that adds a heating system"
-  extends Example5;
+within IDEAS.Examples.Tutorial.DetailedHouse;
+model DetailedHouse6
+  "Extension of DetailedHouse5 that adds a heating system"
+  extends DetailedHouse5;
   package MediumWater = IDEAS.Media.Water "Water Medium";
 
   Fluid.HeatPumps.ScrollWaterToWater heaPum(
```

Comment:
Rename pump instances (e.g. `pumPri` and `pumSec`) and constant for HP control (e.g. `heaPumOn`)